docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -400,27 +418,24 @@ ALPN enables more efficient communication by allowing the client to indicate its
 
 This helps in scenarios like HTTP/2, where multiple protocols might be available, allowing for faster protocol selection.
 
-ALPN is enabled by default.
-To disable it, use the following property:
-
+* ALPN is enabled by default, and it can be disabled by:
```

Comment:
When/why would you disable it? Can it have detrimental performance impact, maybe?

Reply:
Yeah, there are performance reasons, but only experts would disable it (hopefully). Without it, you are going into uncharted territory. 

Again, this guide contains expert-level knowledge. Do not disable ALPN if you do not understand its meaning.

Reply:
Remove the section of your don't think it's useful. 
I can just tell that I had to disable Alpn a few times in various projects for various reasons (native inconsistencies, performance...), and not having to dig into the code to find how to do that can save a lot of time.

Reply:
I will try to integrate this info into it.

Reply:
Warning: Disabling ALPN is not recommended for non-experts, as it can lead to performance degradation, protocol negotiation issues, and unexpected behavior, particularly with protocols like HTTP/2. However, disabling ALPN can be useful for diagnosing native inconsistencies or testing performance in specific edge cases where protocol negotiation causes conflicts.